api/commands/run.ts

```diff
@@ -6,9 +6,34 @@ import * as Long from "long";
 
 const CANCEL_EVENT = "jobCancel";
 
-const isSuccessfulAction = (action: dataform.IExecutedAction) =>
-  action.status === dataform.ActionExecutionStatus.SUCCESSFUL ||
-  action.status == dataform.ActionExecutionStatus.DISABLED;
+const isSuccessfulAction = (actionResult: dataform.IActionResult) =>
+  actionResult.status === dataform.ActionResult.ExecutionStatus.SUCCESSFUL ||
+  actionResult.status === dataform.ActionResult.ExecutionStatus.DISABLED;
+
+const actionExecutionStatusMap = new Map<
+  dataform.ActionResult.ExecutionStatus,
+  dataform.ActionExecutionStatus
+>();
+actionExecutionStatusMap.set(
```

Comment:
This feels like a bad idea. Can we wrap in method or do something like  `new Map<...>([[key, value], [key, value]...])` ?